extensions/git/src/commands.ts

```diff
@@ -1998,7 +1999,9 @@ export class CommandCenter {
 				return;
 			}
 
-			target = choice.label;
+			if (choice instanceof CheckoutItem && choice.refName) {
+				target = choice.refName;
+			}
```

Comment:
Let's add `refName` to all types which are supported in this code path, so we don't have to check for `instanceof`. It's the better general change, ie separate the label from the model.

Reply:
Fixed.